pkg/logs/agent.go

```diff
@@ -49,12 +49,15 @@ func NewAgent(sources *config.LogSources) *Agent {
 
 	// setup the inputs
 	inputs := []restart.Restartable{
-		docker.NewScanner(sources, pipelineProvider, auditor),
 		listener.New(sources, pipelineProvider),
 		file.NewScanner(sources, config.LogsAgent.GetInt("logs_config.open_files_limit"), pipelineProvider, auditor, file.DefaultSleepDuration),
 		journald.New(sources, pipelineProvider, auditor),
 		windowsevent.New(sources, pipelineProvider),
 	}
+	if input, err := container.NewScanner(sources, pipelineProvider, auditor); err == nil {
+		// FIXME: improve input creation
```

Comment:
`improve` is blury, could you add details about what should be changed, or remove the fixme?